src/nimblepkg/packageparser.nim

```diff
@@ -176,10 +117,6 @@ proc validatePackageInfo(pkgInfo: PackageInfo, options: Options) =
     if pkgInfo.backend notin ["c", "cc", "objc", "cpp", "js"]:
       raise validationError("'" & pkgInfo.backend &
           "' is an invalid backend.", false)
-  if options.action.typ in {actionInstall, actionBuild, actionDevelop, actionCompile, actionCheck}:
```

Comment:
Alternatively we can limit the warning to show for only actionCheck, actionDevelop.